test/check/check.go

```diff
@@ -377,6 +402,16 @@ func checkForSecurityContextConstraintsRemoval(name string) error {
 	return isNotFound("SecurityContextConstraints", name, err)
 }
 
+func checkForSecretRemoval(name string) error {
+	err := checkForSecret(name)
```

Comment:
I like the idea, but I hate partial re-factoring. Please open a PR where you adjust all `*Removal` functions to this format.

Reply:
ack